tools/src/main/resources/template/broker.properties

```diff
@@ -0,0 +1,167 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This configuration file is intended for use in KRaft mode, where
+# Apache ZooKeeper is not present.  See config/kraft/README.md for details.
+#
+############################# Server Basics #############################
+# The role of this server. Setting this puts us in KRaft mode
+process.roles=broker
+# The node id associated with this instance's roles
+node.id=2
+# The connect string for the controller quorum
+controller.quorum.voters=1@localhost:9093
```

Comment:
Why are there again three templates? What the difference between "s3shell-kafka-sdk/src/main/resources/template" and "tools/src/main/resources/template"?

Reply:
It is duplicated and I will remove them.